### PR TITLE
Add CLI to the module

### PR DESCRIPTION
**Pull request by icyflame**
- Uses https://github.com/sindresorhus/meow

### CODE REVIEW

**Comment by jonschlinkert** [comment]
thanks for the pr. I'll need to think about this though, since It adds a bunch of dependencies.

**Comment by icyflame** [comment]
Sure. This would be really helpful during testing, which is the reason I added it. 

On second thought, it would be pretty easy to do it without any extra dependencies too. (Using `process.argv`) if you would like me to do it that way, I can change and resubmit this pr.

**Comment by jonschlinkert** [comment]
> it would be pretty easy to do it without any extra dependencies too.

that would be great! I'd be more than happy to merge then, thanks!

**Comment by icyflame** [comment]
Done! I have removed the dependencies, and done it using just `process.argv`. Have a look!

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "type": "git",
     "url": "https://github.com/jonschlinkert/window-size.git"
   },
+	"bin": "cli.js",
```
Comment (jonschlinkert) [inline package.json]: would you mind using 2 spaces to be consistent with the rest of the code? other than that, looks great! thanks!

**Comment by icyflame** [comment]
Fixed!

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "type": "git",
     "url": "https://github.com/jonschlinkert/window-size.git"
   },
+  "bin": "cli.js",
```
Comment (jonschlinkert) [inline package.json]: great! could you also add `cli.js` to the `files` array in package.json? thanks!

**Comment by icyflame** [comment]
Done!

**Comment by jonschlinkert** [comment]
bumped, tagged and published! thanks again!

**Pull request merged**